core_java/twitter/src/main/java/ca/jrvs/apps/twitter/service/TwitterService.java

```diff
@@ -0,0 +1,88 @@
+package ca.jrvs.apps.twitter.service;
+
+import ca.jrvs.apps.twitter.dao.CrdDao;
+import ca.jrvs.apps.twitter.model.Tweet;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@org.springframework.stereotype.Service
+public class TwitterService implements Service {
+
+  private CrdDao dao;
+
+  @Autowired
+  public TwitterService(CrdDao dao) {this.dao = dao;}
+
+  @Override
+  public Tweet postTweet(Tweet tweet) {
+    //Business logic:
+    //e.g. text length, lat/lon range, id format
```

Comment:
remove demo comments. you need to write your own comments. don't just copy and paste